### PR TITLE
Homepage

### PR DESCRIPTION
**Pull request by Sulemana24**
Made changes to footer text

### CODE REVIEW

**Comment by gogurlbaby** [comment]
### Required Changes ♻️

Hi @Sulemana24.

Good job so far!
There are some issues that you still need to work on to go to the next task but you are almost there!

159483175-2f61461d-b352-4cb4-98f8-b173e05ab5ff

### Highlights

✔️ Correct File Structure

### Required

- [ ] _Check the comments under the review._

### Optional Suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me [@gogurlbaby ] in your question so I can receive the notification.

**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**

**Comment by gogurlbaby** [comment]
### Apply These Changes Effectively

### Well done!! Nice Website

### Comments
1. Use comments to describe your work or differentiate sections. 
Eg.
 `<!-- Navbar Section-->`
 `<!-- Hero Section-->`

**Remove the comments you do not need in your work. 
Not a good practice to send multiple comments of a section for review.**

### NavBar Section

2. Nice Navbar. I love how you used the same container for both desktop and mobile nav-items.

- The hover on the **home nav-item** clashes with the white background color of the section on the left. 

- Either give the nav-items more margin on the left or change the hover color for home.

**Hero Section**
 
3. The hero section looks excellent but align the **Play video** text and icon.

### Content Section

**Latest Articles Section**

4. Excellent 💯 

- Add enough padding to the buttons on the section.

- Align the search input and search button.

- No need to use a placeholder called search again with search  as text too on the button. Rephrase it rather. Eg. Search for more articles. Or you can maintain the search placeholder and use a search icon instead of the button.


**Testimonial Section (What Our Visitors Say Section)**

5. It looks good but I suggest you make the section span. 

- Then you can have 2 testimonials cards each for desktop and one card for mobile.

- Consider the content. Read it well and make the necessary corrections, where appropriate.

### Footer Section

6. The footer section looks good but use margin bottom to space out the items on the section. Give them enough breathing space.


### Mobile Section

- Almost all the sections looks good for mobile and tablet.

- The hero section is shifted to the left and the latest articles section is shifted to the right. **Fix them**

### Styles Section

Very good use of the css. Use of variables, class selectors, pesudo-classes and pseudo-elements are all on point. 

**KEEP IT UP** 👍

**Review by gogurlbaby — CHANGES_REQUESTED** [review]
Kindly work on the requested changes ASAP so we move to the next pages.

**Comment by Sulemana24** [comment]
Changes are made pls. Kindly check them out @gogurlbaby

File: home.html [hunk 1]
```diff
@@ -12,6 +12,7 @@
 </head>
 <body>
     <header>
+        <!--Navbar Section-->
```
Comment (gogurlbaby) [inline home.html:15]: Good use of comments in your codes.

File: home.html [hunk 2]
```diff
@@ -12,6 +12,7 @@
 </head>
```
Comment (gogurlbaby) [inline home.html:12]: Add a favicon to your title.

File: home.html [hunk 3]
```diff
@@ -32,24 +33,24 @@
                 </li>
             </ul>
     </nav>
+                <!--Hero Section-->
         <div class="header-content">
                 <h1>The Future of Artificial Intelligence.</h1>
```
Comment (gogurlbaby) [inline home.html:38]: Reduce the font-size of the title for mobile. The hero section for mobile is still shifted to the left. Fix it

File: home.html [hunk 4]
```diff
@@ -12,6 +12,7 @@
 </head>
 <body>
     <header>
+        <!--Navbar Section-->
     <nav>
         <div class="nav-header">
             <div class="nav-logo">
```
Comment (gogurlbaby) [inline home.html:18]: Try using a different color for the hover effect for the nav-items. It should contrast the background-color of contact-us button.

Do not use same class name for id. How do we tell the difference?

File: home.html [hunk 5]
```diff
@@ -71,6 +72,7 @@ <h3 class="section-header">Why you should Start Coding?</h3>
                 <button class="btn">Read more <span><i class="ri-arrow-right-up-line"></i></span></button>
             </p>
         </div>
+        <!--Top Stories Section-->
```
Comment (gogurlbaby) [inline home.html:75]: The background-color of latest articles is visible at the extreme right section of the top stories section. Fix it.

Let the top-stories background color cover the top-stories section.

File: home.html [hunk 6]
```diff
@@ -100,34 +102,38 @@ <h4>
...
+    <h2 class="testi-head">What Our Visitors Say?</h2>
+    <section class="testimonials">
         
-    </section>
-    <section class="testimonials-section">
-        <h2>What Our Visitors Say</h2>
-        <div class="testimonial">
-            <p class="quote">"This blog has helped my child immensely with on his computer projects. Their approach is engaging and effective!"</p>
-            <p class="author">– Sarah L., Parent</p>
+        <div class="testimonial-card">
```
Comment (gogurlbaby) [inline home.html:116]: Very good on applying the changes for this. However, add enough padding for top and bottom for the cards. Overall they look very good.

File: home.html [hunk 7]
```diff
@@ -100,34 +102,38 @@ <h4>
                     Lorem ipsum dolor sit amet consectetur adipisicing elit. Porro assumenda sit non, nam in sint beatae. Veniam, mollitia ratione, explicabo ipsum ab minima nesciunt magnam, quis pariatur asperiores necessitatibus cumque.
                     <button class="btn">Read more <span><i class="ri-arrow-right-up-line"></i></span></button>
                 </p>
-                <div class="search">
-                <input type="text" placeholder="Search...">
-                <button class="search-button">Search</button>
+                <div id="searchBox">
+                    <input type="text" id="input" placeholder="Search our blog...">
```
Comment (gogurlbaby) [inline home.html:106]: Perfect placeholder!

File: home.html [hunk 8]
```diff
@@ -100,34 +102,38 @@ <h4>
...
+        <div class="testimonial-card">
+            <p class="testimonial-text">"Discovery this website is one of my archievements this year, I got free coureses to improve my skills as a student. "</p>
+            <h3 class="testimonial-author">– Sulemana I., Student</h3>
         </div>
     </section>
+
     <div class="search-container">
     </div>
+            <!--Footer Section-->
     <footer class="footer" id="news">
```
Comment (gogurlbaby) [inline home.html:137]: Footer section for mobile is shifted to the left. Fix it.

Input for footer section is too big. Fix it.

File: home.html [hunk 9]
```diff
@@ -32,24 +33,24 @@
...
     </nav>
+                <!--Hero Section-->
         <div class="header-content">
                 <h1>The Future of Artificial Intelligence.</h1>
                 <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Provident veritatis, placeat corporis illum cupiditate odit, accusamus modi.consectetur adipisicing elit.</p>
             <div class="header-btns">
                 <button class="btn">Read more <span><i class="ri-arrow-right-up-line"></i></span></button>
                 <button class="btn">
-                    <a href="https://www.youtube.com/watch?v=SJNbnVjecf0" target="_blank">Play video <span><i class="ri-play-circle-fill"></i></span></a>
+                    <a href="https://www.youtube.com/watch?v=dv9q7Ema40k" target="_blank">Play video <span><i class="ri-play-circle-fill"></i></span></a>
```
Comment (gogurlbaby) [inline home.html:43]: Looks good but add enough gap between the text and icon. In addition, capitalise the text.

File: home.html [hunk 10]
```diff
@@ -169,75 +175,9 @@ <h2>Join our newsletter</h2>
             </div>
         </div>
         <div class="footer-bar">
-            Copyright <i class="ri-copyright-line"></i> 2024 Bindawdi Enterprise. All rights reserved.
+            Copyright <i class="ri-copyright-line"></i> 2024 Bindawdi Enterprise.
```
Comment (gogurlbaby) [inline home.html:178]: It seems the green background-color is over-powering the white color text. Try tweaking the text color a bit to be very visible.

File: home.html [hunk 11]
```diff
@@ -100,34 +102,38 @@ <h4>
...
+                <div id="searchBox">
+                    <input type="text" id="input" placeholder="Search our blog...">
+                    <button id="searchBtn">Search</button>
                 </div>
             </div>
         </div>
     </div>
     </section>
-    <section>
+    <h2 class="testi-head">What Our Visitors Say?</h2>
```
Comment (gogurlbaby) [inline home.html:113]: Reduce font-size for mobile.

**Comment by gogurlbaby** [comment]
### Responsiveness

- _A TIP: Start styling your page for mobile first and then use media queries to make it responsive for desktop and tablet._  

**That way you make things easy for yourself.**

- _So inspect your page and start styling for mobile first_ 

### New Branch
Create a new branch called **fix-homepage** and push your new changes there.

After making all the new changes, remember to create the new branch before adding and committing the new changes. Then push to the new branch.


### **SUBMISSION: FRIDAY**

**Immediately reach out if you are not clear about something**

**All the best!!**

**Review by gogurlbaby — CHANGES_REQUESTED** [review]
Create a new branch called fix-homepage and push your new changes there.

After making all the new changes, remember to create the new branch before adding and committing the new changes. Then push to the new branch.

**Comment by Sulemana24** [comment]
New branch has been created. And all changes has been made, pls check on it @gogurlbaby

**Pull request merged**